app/models/tag.rb

```diff
@@ -23,6 +23,8 @@ class Tag < ActsAsTaggableOn::Tag
   mount_uploader :profile_image, ProfileImageUploader
   mount_uploader :social_image, ProfileImageUploader
 
+  validates :name,
+            format: /\A[A-Za-z0-9\s]+\z/, allow_nil: true
```

Comment:
There are some specs elsewhere within the codebase that assert the creation of tags with whitespace in `Tag.name`. Highly doubt this is a practical issue, as you can't enter whitespace in the tag input in the client.

LMK if I should clamp this down some more by removing the `\s` and updating the aforementioned specs!